src/packages/server/utils/set-cors-headers.js

```diff
@@ -0,0 +1,22 @@
+import type { Response } from '../response';
+import type { Server$cors } from '../interfaces';
+
+export default function setCORSHeaders(res: Response, {
+  origin,
+  methods,
+  headers,
+  enabled
+}: Server$cors) {
+  if (!enabled) return;
```

Comment:
I'm surprised we don't have a lint rule for this but we should prefer multi-line block `if statements`:

```javascript
  if (!enabled) {
    return;
  }
```

--

The reason for this is to make it easier to add to this logic in the future.